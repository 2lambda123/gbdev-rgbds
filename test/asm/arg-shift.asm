print_all: MACRO
    REPT _NARG
    	PRINTT " \1"
    	SHIFT
    ENDR
    PRINTT "\n"
ENDM

print_some: MACRO
    PRINTT "\1"
    SHIFT 5
    PRINTT "\2\6\9"
    SHIFT 17
    SHIFT
    PRINTT "\3\9"
ENDM


	print_all This test, probably, passes\,, but who knows, ?

	print_some R,e,d,n,e,x,G,a,m,e,B,o,y,D,e,v,e,l,o,p,e,m,e,n,t,S,y,s,t,e,m,\n
