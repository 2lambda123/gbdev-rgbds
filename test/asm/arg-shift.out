 This test probably passes, but who knows ?
RGBDS
